Sources/SwiftProtobuf/Message.swift

```diff
@@ -89,21 +125,28 @@ public protocol Message: CustomDebugStringConvertible {
   // google.protobuf.Any support
   //
 
-  // Decode from an `Any` (which might itself have been decoded from JSON,
-  // protobuf, or another `Any`).
+  /// Decode from an instance of `Any` (which might itself have
+  /// been decoded from JSON, protobuf, or another `Any`).
+  ///
+  /// - Parameter any: item to decode
+  /// - Throws: an instance of `DecodingError` if the message cannot be parsed
   init(any: Google_Protobuf_Any) throws
 
   /// Serialize as an `Any` object in JSON format.
   ///
   /// For generated message types, this generates the same JSON object as
   /// `serializeJSON()` except it adds an additional `@type` field.
```

Comment:
More explanation:  This is used by the JSON serialization machinery to allow certain types to override how they are represented within `Any` containers.  You should not normally call or override this method.